test/integration/default/inspec/default_spec.rb

```diff
@@ -7,3 +7,24 @@
     it { should be_installed }
   end
 end
+
+describe file('/usr/local/bin/phpcheck') do
+  it { should exist }
+  its('content') { should match %r{find . -name '*.php' -or -name '*.ini' | grep -v '/geshi/' | xargs grep -H -T -e "$*" | cut -d/ -f2 | uniq} }
+end
+
+describe file('/usr/local/bin/phpshow') do
+  it { should exist }
+  its('content') { should match %r{find . -name '*.php' -or -name '*.ini' | grep -v '/geshi/' | xargs grep --color=auto -H -T -e "$*"} }
+end
+
+# inspec does not handle chaining commands nor bash -c "$cmds"
+# describe command('cd /var/www; phpcheck test for me') do
```

Comment:
I think given the suggestion I made above, this should work without requiring being in the directory you need to be in.